src/components/userauth/ErrorMessage/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ErrorMessage snapshot test 1`] = `null`;
```

Comment:
this test just says null.

add a second test where you pass an error message please

Reply:
I didn't understand what you meant by second test but I passed a test message as prop to the test component. Now snapshot is not null

Reply:
you are allowed to have more than one test in a file. and you should most of the time, especially if there are two cases

Reply:
So I should be displaying all conditions inside snapshot. I get it.